services/QuillLMS/app/services/quill_big_query/materialized_view_refresher.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module QuillBigQuery
+  class MaterializedViewRefresher < ApplicationService
+    attr_reader :query_key
+
+    QUERY_FOLDER = Rails.root.join('db/big_query/views/')
+    CONFIG = Configs[:big_query_views]
```

Comment:
Is there a reason to use the `Evidence::Configs` initializer here? Seems like an unnecessary indirection. How about `BigQuery::Configs`?

Reply:
I think there might be some confusion here. `configs` is a [small gem](https://github.com/kickstarter/configs) that is helpful for querying `yaml` files. This isn't a reference to Evidence::Configs but the Configs gem.

Reply:
Got it, carry on